### PR TITLE
GEOMESA-1531 Kafka tools for 09, 10

### PR DESCRIPTION
**Pull request by elahrvivaz**
* Modules are copy-pasted from 08 tools

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
All new files are just copies of existing 08 tools with imports tweaked

File: geomesa-kafka/geomesa-kafka-tools/bin/install-kafka.sh [hunk 1]
```diff
@@ -7,43 +7,39 @@
...
+  elif [[ $kafka_major_version = "10" ]]; then
+    ZK_CLIENT_VERSION="0.8"
   else
-    echo "Error: Kafka version ${kafka_major_version} is not supported.  Please select 0.9 as second arg or omit to use default (0.8)."
+    echo "Error: Kafka version ${kafka_major_version} is not supported."
     exit
   fi
 
+  declare -a urls=(
+    "https://search.maven.org/remotecontent?filepath=org/apache/kafka/kafka-clients/%%kafka.min.version%%/kafka-clients-%%kafka.min.version%%.jar"
```
Comment (jahhulbert-ccri) [inline geomesa-kafka/geomesa-kafka-tools/bin/install-kafka.sh]: would you mind pulling out the URL up top like I think we do in one of the other scripts? helps for when we have to install it in places with no internets.

Comment (elahrvivaz) [inline geomesa-kafka/geomesa-kafka-tools/bin/install-kafka.sh]: fixed

**Comment by Afterglow375** [comment]
Tested this out, worked fine :+1:

**Pull request merged**